.github/workflows/stale-bot.yaml

```diff
@@ -0,0 +1,28 @@
+# Mark PRs and Issues as stale after a given time. 
+# Closes stale issues but not stale PRs.
+
+name: 'Mark stale issues and PR'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not sure if `getporter` has this but using this, this token would have to be set.

Reply:
Hey @troy0820 thanks for the review. I took this from another yaml file in this project https://github.com/getporter/porter/blob/be5c66a99386a347ff90e72c77e02122cff62cd8/.github/workflows/close-issues.yaml#L19 Not sure if it's set. Is it possible to confirm if it's indeed set?

Reply:
^^ was gonna suggest that it be this